src/main/java/org/spongepowered/api/util/command/dispatcher/Dispatcher.java

```diff
@@ -49,30 +54,33 @@
     /**
      * Get a list of primary aliases.
      *
-     * <p>The returned collection cannot be modified.</p>
+     * <p>
+     * The returned collection cannot be modified.
```

Comment:
This is no longer a list/collection now it's a Map. The javadocs need updating and please explain what the keys/values represent.